Update tac-member-R&Rs.md (#399)
ossf/tac: process/tac-member-R&Rs.md

- Establish and drive an overall Technical Vision aligned with the Foundation's MVSR for the community 
- Regularly attend TAC meetings, participate in discussions, voting, and ad-hoc meetings as needed to support Technical Initiatives
- Approve, establish, structure, organize, and archive Technical Initiatives and associated policies and procedures; identify gaps that require new Technical Initatives and advocate to close those gaps in the open source community; if a need for shared services that benefits stakeholders is identified by the community, communicate with the governing board to evaluate services on a case-by-case basis

> - Review and engage on open Github Issues and Pull Requests in advance of upcoming TAC meetings. Prior to each TAC meeting, review open pull requests, with the goal to approve or provide active comments and feedback preventing approval. Between TAC meetings comment on open Issues, seeking to drive Issue resolution on behalf of the community for open Issues in a timely manner.  

Reviewer: I think there are couple of pieces that are being repeating here. How about the following?
```suggestion
- Review and engage on open Github Issues and Pull Requests in advance of upcoming TAC meetings, with the goal to approve or provide active comments and feedback preventing approval at each meeting. Between TAC meetings comment on open Issues, seeking to drive Issue resolution on behalf of the community in a timely manner.  
```

Reviewer 2: +1 to these wording changes. I also suggest splitting these into sub-bullets for better legibility.
```suggestion
- Review and engage on open Github Issues and Pull Requests in advance of upcoming TAC meetings:
   - Prior to each TAC meeting: with the goal to approve or provide active comments and feedback preventing approval. 
   - Between TAC meetings: seeking to drive Issue resolution on behalf of the community for open Issues in a timely manner.  
```
